rust/operator-binary/src/zk_controller.rs

```diff
@@ -88,6 +89,10 @@ pub enum Error {
         source: strum::ParseError,
         role: String,
     },
+    #[snafu(display("internal operator failure: {source}"))]
```

Comment:
No need to mention the source in the message.
```suggestion
    #[snafu(display("internal operator failure"))]
```